qiskit/circuit/library/standard_gates/x.py

```diff
@@ -1416,6 +1418,8 @@ def __init__(
         duration=None,
         unit="dt",
         _base_label=None,
+        relative_phase: bool = False,
+        action_only: bool = False,
```

Comment:
Could you please add proper documentation on all of these parameters so that the user will know what they are doing and what to choose?
It will be good to extend the docstring of the `MCXVChain` class, add a reference to the relevant paper and theorem.
Perhaps even some examples if possible?

Reply:
It seems that there are 3 boolean parameters: 
`dirty_ancillas,  relative_phase, action_only`
Could we have all 8 options for these parameters? or only part of the options are possible?
If so, please add it to the docs, check and raise a `QiskitError` if the user chose a bad option.
Otherwise, please add tests for all relevant options.

Reply:
Are there some bounds on the number of `CX` gates for these options? 
If so, could you please add this to the docs, as well as add proper tests.